src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -1,9 +1,104 @@
 import math
-from typing import List, Optional, Tuple
+from typing import List, Optional, Tuple, Union
 import numpy as np
 from scipy.sparse import coo_matrix
 from ophys_etl.extractors.motion_correction import MotionBorder
 from ophys_etl.types import DenseROI, ExtractROI
+from skimage.morphology import binary_opening, binary_closing, disk
+
+
+def full_mask_constructor(mask_matrix: List[List[bool]], x: int, y: int,
+                          shape: Tuple[int, int]) -> np.ndarray:
+    """make a full-framed mask of the ROI
```

Comment:
I would maybe improve this so that there won't be confusion from future developers...

Maybe add a note like:

```
DenseROI masks are generated by saving only the smallest bounding box that contains the ROI as a dense array. This function takes the bounded ROI as well as as the x and y top left position of the bounding box and regenerates the ROI as it would appear in the original field of view
```

Reply:
added note